pkg/cmd/ssh.go

```diff
@@ -15,40 +15,43 @@
 package cmd
 
 import (
-	"encoding/json"
 	"errors"
 	"fmt"
 	"io/ioutil"
 	"os"
 	"path/filepath"
-	"strings"
 
 	gardencorev1alpha1 "github.com/gardener/gardener/pkg/apis/core/v1alpha1"
-	v1alpha1 "github.com/gardener/gardener/pkg/client/core/clientset/versioned/typed/core/v1alpha1"
-	"github.com/jmoiron/jsonq"
 	"github.com/spf13/cobra"
 	v1 "k8s.io/api/core/v1"
 	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
-	//BastionOSName is the name of the operation system used for creating bastion
+	// BastionOSName is the name of the operation system used for creating bastion
 	BastionOSName = "coreos"
 
-	//ControllerRegistrationAwsName is name of controller registration for AWS
+	// ControllerRegistrationAwsName is name of controller registration for AWS
 	ControllerRegistrationAwsName = "provider-aws"
 
-	//NodeRegionLabel is the label name for the node region
+	// NodeRegionLabel is the label name for the node region
 	NodeRegionLabel = "failure-domain.beta.kubernetes.io/region"
+
+	// WarningColor prints user warnings in red
+	WarningColor = "\033[1;31m%s\033[0m"
 )
 
+// OSUsers contains all supported OS users
+var OSUsers = []string{"core", "ubuntu", "ec2-user"}
```

Comment:
yes, `gardener` user is already merged in the gardener https://github.com/gardener/gardener/pull/1513
SSH keys are on its way to be merged, too https://github.com/gardener/gardener-extensions/pull/372.
So @DockToFuture , please use only `gardener` user for login on the nodes.

Reply:
Yup, You can now check if the `gardener` user exists in the node and just use it for the` bastion` and the ssh. Otherwise you can ask the user to provide `ssh-user` with flags or just print the command with placeholders like before :)